lib/BloqadeLattices/test/bounded_lattice.jl

```diff
@@ -1,39 +1,51 @@
 using Test
 using BloqadeLattices
 
-# chain
-bounded_lattice = parallelepiped_region(ChainLattice(),(4,);pbc=true)
-expected_positions = [(0.0,),(1.0,),(2.0,),(3.0,)]
+# catch invalid scale (must be greater than 0)
+negative_scale = -0.1
+# define chain lattice
+@test_throws "scale must be a positive real value." parallelepiped_region(ChainLattice(),(4,);pbc=true,scale=negative_scale)
+
+zero_scale = 0.0
+@test_throws "scale must be a positive real value." parallelepiped_region(ChainLattice(),(4,);pbc=true,scale=zero_scale)
+
+scale = π
+bounded_lattice = parallelepiped_region(ChainLattice(),(4,);pbc=true,scale=scale)
+expected_positions = [(0.0*scale,),(1.0*scale,),(2.0*scale,),(3.0*scale,)]
 @test issetequal(bounded_lattice.site_positions,expected_positions)
 @test dimension(bounded_lattice) == 1
 
 # square
-bounded_lattice = parallelepiped_region(SquareLattice(),(2,0),(0,2);pbc=true)
-expected_positions = [(0.0,0.0),(1.0,0.0),(0.0,1.0),(1.0,1.0)]
+bounded_lattice = parallelepiped_region(SquareLattice(),(2,0),(0,2);pbc=true,scale=scale)
+expected_positions = [(0.0*scale,0.0*scale),(1.0*scale,0.0*scale),(0.0*scale,1.0*scale),(1.0*scale,1.0*scale)]
```

Comment:
I think it would prevent future headaches and reduce fragile code if we avoid having to type `scale` over and over 😅 

```suggestion
base_expected_positions = [(0.0,0.0),(1.0,0.0),(0.0,1.0),(1.0,1.0)]
scaled_expected_positions = map(base_expected_positions) do pos
    scale .* pos
end
```

Reply:
This should work with other 2D and 3D cases (such as the kagome instance)